utilities/BaseClass.py

```diff
@@ -0,0 +1,141 @@
+import logging
+import os.path
+import time
+from traceback import print_stack
+
+import allure
+from allure_commons.types import AttachmentType
+from selenium.common.exceptions import *
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+import utilities.CustomeLogger as cl
+
+
+class BaseClass:
+    log = cl.custom_logger(logging.DEBUG)
+
+    def __init__(self, driver):
+        self.driver = driver
+
```

Comment:
Please add comment on each function of this file for better understanding in future

Reply:
already added